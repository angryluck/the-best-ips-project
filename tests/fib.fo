fun int fibo(int n) =
  if n == 0 then 0
  else if n == 1 then 1
  else fibo(n - 1) + fibo(n - 2)

fun int fib_case() =
  let n = read(int) in
  let r = fibo(n) in
  write(r)

fun int main() =
  let tmp = fib_case() in
  let tmp = fib_case() in
  let tmp = fib_case() in
  let tmp = fib_case() in
  let tmp = fib_case() in
  let tmp = fib_case() in
  let tmp = fib_case() in
  fib_case()