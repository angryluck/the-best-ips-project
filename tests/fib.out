0
1
1
2
5
55
610
6765
